### DISCUSSION TITLE
Resource discovery documentation

### DISCUSSION THREAD

**Discussion by ovkhasch** [tomkerkhove/promitor#1047] [General] [answered]
I was looking for any documentation on resource discovery feature without any success. It would be nice to know how is it possible to use scraper with resource discovery to scrape for all the resources in a subscription without need to list all the resource groups in a config.

**Comment by tomkerkhove** [comment, marked as answer]
Resource discovery is still WIP and currently tracked via #512, feel free to subscribe there!

It should be available as of Promitor Scraper v1.7

**Reply by ovkhasch** [reply]
Thanks for a quick response! Is there a planned release date for 1.7?

**Reply by tomkerkhove** [reply]
ASAP 😁

I'm hoping to land this in June but it requires some big changes so I can't promise anything. Current config looks like [this](https://github.com/tomkerkhove/promitor/blob/master/config/promitor/resource-discovery/resource-discovery-declaration.yaml) for resource discovery declaration
